packages/demo/src/container/index.tsx

```diff
@@ -1,11 +1,12 @@
-import { Editor, EditorProps } from '@edtr-io/core'
```

Comment:
Review that and use EditorProvider instead. Also note how the API feels in both cases (and improve it if needed)